docs/CHANGES.rst

```diff
@@ -4,6 +4,19 @@ Changelog
 4.1.1 (unreleased)
 ------------------
 
+- Ported atomic updates from ftw.solr.
```

Comment:
@maethu wouldn't it make sense to add this to our new sphinx-based docs folder and just link to that here?